src/rinoh/image.py

```diff
@@ -253,11 +254,14 @@ def render(self, container, last_descender, state, **kwargs):
             filename_or_file = self._absolute_path_or_file()
             image = container.document.backend.Image(filename_or_file)
         except OSError as err:
-            container.document.error = True
             message = "Error opening image file: {}".format(err)
             self.warn(message)
-            error = ErrorText(message)
-            return Paragraph(error).flow(container, last_descender)
+            if self.alt is None:
+                container.document.error = True
+                error = ErrorText(message)
+                return Paragraph(error).flow(container, last_descender)
+            alt_text = MixedStyledText(self.alt)
+            return Paragraph(alt_text).flow(container, last_descender)
```

Comment:
This Paragraph should be selectable in the matcher/style sheet so that it can be styled. Something like:
```python
Paragraph(alt_text, style='image alt')
```

Same goes for the original ErrorText, actually. This is now given a fixed style (`font_color=RED`)

Reply:
I've added your line and also added the style to the matcher.